packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/videos.blade.php

```diff
@@ -1,5 +1,9 @@
 {!! view_render_event('bagisto.admin.catalog.product.edit_form_accordian.videos.before', ['product' => $product]) !!}
 
+@php
+    $max_upload = ini_get("upload_max_filesize");
```

Comment:
$maxUpload should be in camel case

Reply:
I think this is not a good place to declare the max constant variable. This can be placed in core class as a constant and can be use as,

`core()->getMaxUploadSize()`

Something like this.